ps_emailalerts.php

```diff
@@ -609,21 +609,33 @@ public function hookActionUpdateQuantity($params)
             }
         }
 
-        if ($this->customer_qty && $quantity > 0) {
-            MailAlert::sendCustomerAlert((int) $product->id, (int) $params['id_product_attribute']);
+        if ($product_has_attributes) {
+            $sql = 'SELECT `minimal_quantity`, `id_product_attribute`
+                FROM ' . _DB_PREFIX_ . 'product_attribute
+                WHERE id_product_attribute = '. $id_product_attribute;
+
+            $result = Db::getInstance()->getRow($sql);
+
+            if (isset($result['minimal_quantity']) && $this->customer_qty && $quantity >= $result['minimal_quantity']) {
```

Comment:
```suggestion
            if ($result && $this->customer_qty && $quantity >= $result['minimal_quantity']) {
```

Reply:
Thank i made a commit to also add in hookActionProductAttributeUpdate